source/agora/consensus/EnrollmentManager.d

```diff
@@ -525,6 +554,11 @@ unittest
     assert(man.addEnrollment(enroll2));
     assert(man.getEnrollmentLength() == 2);
 
+    Enrollment[] enrolls;
+    man.getEnrollments(enrolls);
+    assert(enrolls.length == 2);
+    assert(enrolls[0].utxo_key < enrolls[1].utxo_key);
```

Comment:
I think a full test would include:
- 3 enrollments, added in (utxo key) order;
- 3 enrollments, added in reverse order;

At the moment the sortedness is not really tested. It should also be tested on the block validation level, but maybe you want to keep this for another PR ?

Reply:
Yes, I'll add a test at the block level in another PR to be added.